src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/AppContext.java

```diff
@@ -39,10 +40,12 @@
   // initiate a node config cache within each AppContext space
   // to store node config settings from ES
   private final NodeConfigCache nodeConfigCache;
+  private Set<String> mutedActions;
```

Comment:
The RCAController thread updates this but it is read by the SchedulerTask threads. We should make this `volatile`

Reply:
Done.